packages/RadioGroup/src/Variant.ts

```diff
@@ -0,0 +1,6 @@
+enum Variant {
```

Comment:
Few thoughts here – I ran into a case where I needed to do something similar for Syntax.

1. Some packages also have `Size` which is treated similarly. Should we name this file to something more generic for other enums (or whatever we decide on), and make `Variant` a named export to extend this pattern across packages?
2. Haven't we been naming these enums `Variants`?
3. We should probably figure out the right pattern for these, since it sounds like we probably aren't going forward with enums for this case.

Reply:
I don't _think_ I've been using the plural. In general I try not to, the type is defining what a valid single value is, so the singular feels more accurate.

Reply:
I think you're right – my bad.